syntax="proto2";

import "aap_protobuf/service/control/message/NavFocusType.proto";

package aap_protobuf.service.control.message;

message NavFocusRequestNotification {
    optional NavFocusType focus_type = 1;
}